src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportControllerTest.java

```diff
@@ -143,7 +139,6 @@ public void setUp() {
             intentionToProceedService
         );
         sampleClaim = SampleClaim.getDefault();
-        when(userService.authenticateAnonymousCaseWorker()).thenReturn(USER);
     }
 
     @Test
```

Comment:
looks like you can set test name to be whatever you want as display name but i think its fine without it

Reply:
`@DisplayName()` becomes very useful with `@NestedClass`, which this test class actually lends itself to quite nicely! If we grouped (nested) tests by the original method then display names - at least on the classes - would make the results much more understandable. It's not _needed_, but it would help to showcase one of the benefits of JUnit 5